src/single-spa.js

```diff
@@ -1,7 +1,7 @@
 export { start } from './start.js';
 export { ensureJQuerySupport } from './jquery-support.js';
-export { setBootstrapMaxTime, setMountMaxTime, setUnmountMaxTime, setUnloadMaxTime } from './child-applications/timeouts.js';
-export { declareChildApplication, getMountedApps, getAppStatus, unloadChildApplication, checkActivityFunctions, getAppNames } from './child-applications/child-apps.js';
+export { setBootstrapMaxTime, setMountMaxTime, setUnmountMaxTime, setUnloadMaxTime } from './applications/timeouts.js';
+export { registerApplication, getMountedApps, getAppStatus, unloadApplication, checkActivityFunctions, getAppNames } from './applications/apps.js';
```

Comment:
This is a breaking change. If we want to keep this as a 3.7.0 release, we should export declareChildApplication and unloadChildApplication as well as the new ones. Maybe with deprecation notices.

The other option is to release this as 4.0.0.

Reply:
Yeah. I didn't think it was ready to merge into master 100% right now. There is a checklist of items that I think need to still be done before release.